### PR TITLE
Style archive product

### PR DESCRIPTION
**Pull request by mikeyrayvon**

### CODE REVIEW

File: archive-product.php [hunk 1]
```diff
@@ -11,33 +11,31 @@
...
-  $i = 0;
   while( have_posts() ) {
     the_post();
 
     $product = new WC_Product($post->ID);
     $in_stock = $product->is_in_stock();
     $availability = $product->get_availability();
 
-    if ($i % 4 === 0 && $i !== 0) {
-      echo '</div>\n<div class="row margin-bottom-small">';
```
Comment (varffs) [inline archive-product.php:23]: why did you delete this? this is important to stop flow layout breaking? or does flex deal with it in a better way?

File: archive-product.php [hunk 2]
```diff
@@ -11,33 +11,31 @@
...
       <a href="<?php the_permalink() ?>">
         <div class="shop-product-title-holder">
           <h3 class="shop-product-title font-serif font-italic">
             <?php the_title(); ?>
           </h3>
         </div>
         <?php if (!$in_stock) { ?>
-          <span class="shop-product-sold font-uppercase"><?php echo $availability['availability']; ?></span>
+          <span class="row align-center shop-product-sold font-uppercase">
+            <img src="<?php echo get_stylesheet_directory_uri() . '/img/dist/teardrop.svg'; ?>">
```
Comment (varffs) [inline archive-product.php]: you can include the SVG inline via `file_get_contents` to save a request. we dont have a threshold for usage vs size on this but i imagine better to inline but up to you

File: css/site.styl [hunk 3]
```diff
@@ -378,8 +379,18 @@ img
   position: absolute
   top: 0
   left: 0
-  padding: $margin-small
-  z-index: 10
+  padding: $margin-tiny
+  margin-left: $margin-tiny
+  z-index: 100
```
Comment (varffs) [inline css/site.styl]: why did u up the stack on this?

File: css/site.styl [hunk 4]
```diff
@@ -378,8 +379,18 @@ img
...
+  color: $color-white
+  img
```
Comment (varffs) [inline css/site.styl]: I think it would be better to give the image a class and target directly. That way you could also include the hover styles from line 358 and it would be more readable as well as performant

File: archive-product.php (hunk 1 above)
Comment (mikeyrayvon) [inline archive-product.php:23]: because on tablet its 3 to a row and on mobile its 1. we can just let flex wrap the items and use the margin-bottom on the items instead of the row

File: css/site.styl (hunk 3 above)
Comment (mikeyrayvon) [inline css/site.styl]: oh for the hoverstate but then i moved the hover to the container so i can take this off

File: archive-product.php (hunk 1 above)
Comment (varffs) [inline archive-product.php:23]: ok sweet flex deals with the flow thing nicely:

screen shot 2016-08-12 at 16 35 54

File: css/site.styl [hunk 5]
```diff
@@ -354,6 +354,9 @@ img
   img
     display: block
     width: 100%
+  &:hover 
+    .shop-product-title-holder, .shop-product-sold-text
```
Comment (varffs) [inline css/site.styl:358]: I think this hover should go all the way to the parent `article` `.shop-product` otherwise on every img hover on the site the browser is looking for 2 child classes and generally not finding them

Comment (mikeyrayvon) [inline css/site.styl:358]: the hover isnt on `img` its on `.shop-product`.  do you even stylus bro?

**Comment by mikeyrayvon** [comment]
can we close this?

File: css/site.styl (hunk 5 above)
Comment (varffs) [inline css/site.styl:358]: ahh ffs github changed view didnt show me the parent of the img
